backend/services/mgmt/v1alpha1/transformers-service/transformers_test.go

```diff
@@ -39,7 +39,7 @@ func Test_GetSystemTransformers(t *testing.T) {
 
 	assert.NoError(t, err)
 	assert.NotNil(t, resp)
-	assert.Equal(t, 37, len(resp.Msg.GetTransformers())) // the number of system transformers
+	assert.Equal(t, 38, len(resp.Msg.GetTransformers())) // the number of system transformers
```

Comment:
Is this a valid test case? It's going to fail any time we add or remove one.

At best this should just assert that `GetTransformers` does not return an empty list if you want to ensure that it always returns some.

Reply:
update